ZAPD/CrashHandler.cpp

```diff
@@ -0,0 +1,146 @@
+#include "CrashHandler.h"
+#include "Utils/StringHelper.h"
+
+#if defined(__linux__)
```

Comment:
This ought to still work in a POSIX environment, right? Seems to work on Mac, which does not define `__linux__` so there's surely a better condition to use here.